### DISCUSSION TITLE
Fresh installed without user

### DISCUSSION THREAD

**Discussion by ranrinc** [dani-garcia/vaultwarden#3526] [Q&A]
Hi, I just trying the latest vaultwarden to be installed on my fresh Debian server. And previously already successfully install an old version of vaultwarden. Now the. installation with docker successfully establish however, this time the difference is the first request of making the 1st account is not being asked. It goes straight to username/password. Without allow to register and without the admin panel how can I create the 1st account? or did I make some stupid mistake?

**Comment by BlackDex** [comment]
Please provide your docker config so we can help a bit better.
The web-vault provides a link to register an account, it should be visible on the login screen.

![Screenshot_20230513-133323](https://github.com/dani-garcia/vaultwarden/assets/610450/446206c8-9e39-4092-a423-1b7bec6f824d)

**Comment by rakuri255** [comment]
Make sure you are using SSL (HTTPS). 
I had similar issues with unsecure (HTTP). It dont lets you login or create accounts.
